packages/gatsby-cli/README.md

```diff
@@ -23,15 +23,13 @@ We can now use the `develop` script tied to our app's version of Gatsby, rather
 
 ## How to use
 
-Rather than installing the Gatsby CLI tool locally you can run it using [npx](https://www.npmjs.com/package/npx)
-
-To see the commands available to run `npx gatsby --help`.
+Run `gatsby --help` for full help.
```

Comment:
Same here, e.g. reference that it needs to be globally installed and _how_ to globally install.

Reply:
Same here